llvm/test/CodeGen/NVPTX/load-store.ll

```diff
@@ -4,7 +4,7 @@
 ; RUN: llc < %s -march=nvptx64 -mcpu=sm_70 -mattr=+ptx82 | FileCheck %s -check-prefixes=CHECK,SM70
 ; RUN: %if ptxas-12.2 %{ llc < %s -march=nvptx64 -mcpu=sm_70 -mattr=+ptx82 | %ptxas-verify -arch=sm_70 %}
 
-; TODO: add i1, <8 x i8>, and <6 x i8> vector tests.
+; TODO: add i1, and <6 x i8> vector tests.
```

Comment:
This file has grown to be pretty large. Perhaps we should consider splitting some of the tests into a separate file.

Reply:
That seems reasonable. Any suggestions for how to split it?

Reply:
Scalars vs vectors would be one factor.
If vectors test is still too large, it could be split by element type/size (e.g. native vectors vs the ones with small elements that get lowered as b32)

Reply:
Done. I tried to keep only the relevant TODOs at the top of each new file, can you double check my work there?